library/src/window_impl.cxx

```diff
@@ -378,9 +381,13 @@ void window_impl::UpdateDynamicOptions()
   renderer->UpdateLights();
 
   const options& opt = this->Internals->Options;
-  renderer->ShowAxis(opt.ui.axis);
-  renderer->SetUseTrackball(opt.interactor.trackball);
-  renderer->SetInvertZoom(opt.interactor.invert_zoom);
+
+  if (this->Internals->Interactor)
+  {
+    renderer->ShowAxis(opt.ui.axis);
```

Comment:
should we add a warning if `opt.ui.axis` is true but there is no interactor?

Reply:
We definitely can, but then isnt there many options with "no effect" where we should warn as well. eg: `ui.filename` without `ui.filename_info` set.

Reply:
ok fair enough